content/zh/docs/helm/helm_install.md

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
This is an auto-generated file. We need to figure out the best way to handle this; for now, can you put it in a different pull request, so we can merge the normal docs files without having to wait?